DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/camera/ffmpeg.md

The [`ffmpeg` program](https://ffmpeg.org/) program must be installed separately from `viam-server` on your system for this driver to work.
{{< /alert >}}

> First, connect your camera to your machine's computer and turn it on.

Reviewer: ```suggestion
First, connect your camera to your machine's computer and power both on.
```
